bodhi/server/services/updates.py

```diff
@@ -308,8 +309,15 @@ def query_updates(request):
         query = query.filter(Update.date_pushed < pushed_before)
 
     releases = data.get('releases')
+    current_release = data.get('current_release')
     if releases is not None:
         query = query.filter(or_(*[Update.release == r for r in releases]))
+    elif str(current_release).lower() not in ['0', 'false']:
+        query = query.filter(
+            Update.release_id == Release.id
+        ).filter(
+            Release.state.in_([ReleaseState.current, ReleaseState.pending])
```

Comment:
Since pending is also included (and I think reasonably so), I think we really should use the word 'active' instead of 'current'.